### ISSUE TITLE
SA1009 conflicting with SA1024

### ISSUE THREAD

**Issue by AgentFire** [DotNetAnalyzers/StyleCopAnalyzers#3621]
In code like this:

```cs
public sealed record RecordName(int ArgumentName) : IAmInterface
```

I get conflict of SA1024 (Colon should not be preceded by a space) with SA1009 (Closing parentheses should not be followed by a space)

Please fix so that C# Records would be analyzed differently since they may have both parentheses and inheritance.

**Comment by bjornhellander** [comment]
I don't see either of those warnings for that code. What version are you using? 1.1.118 pre-dates this language update, if that is what you are using. You need one of the 1.2 beta versions.

**Comment by AgentFire** [comment]
1.2 betas have no issues. Should I close the issue then?

**Comment by sharwell** [comment]
Duplicate of #3248
